src/www/ui/template/ui-browse.js.twig

```diff
@@ -62,7 +62,7 @@ $(document).ready(function() {
   browseTable = createBrowseTable();
   $('.clickable-folder').click(function(event){
     folder = $(this).attr('data-folder');
-    browseTable.fnDraw();
+    browseTable.Draw();
```

Comment:
I am sorry, it is supposed to be `draw()`
https://datatables.net/reference/api/draw()